build.gradle

```diff
@@ -2,6 +2,7 @@
 buildscript {
 	repositories {
 		mavenCentral()
+        mavenLocal()
```

Comment:
You always seem to have problems getting the indentation right :-)

I really recommend making the white space characters visible in your editor, then you never have these kind of issues.